### ISSUE TITLE
Bug report

### ISSUE THREAD

**Issue by Rthvk** [BhavyeMathur/goopylib#81]
```python
Traceback (most recent call last):
  File "C:/Users/rithv/PycharmProjects/PyhtonPrograms/Hello_World!.py", line 1, in <module>
    from goopylib.imports import *
  File "C:\Users\rithv\PycharmProjects\PyhtonPrograms\venv\lib\site-packages\goopylib\__init__.py", line 2, in <module>
    from goopylib import imports
  File "C:\Users\rithv\PycharmProjects\PyhtonPrograms\venv\lib\site-packages\goopylib\imports.py", line 1, in <module>
    from goopylib.styles import *
  File "C:\Users\rithv\PycharmProjects\PyhtonPrograms\venv\lib\site-packages\goopylib\styles.py", line 1, in <module>
    from goopylib.colours import *
  File "C:\Users\rithv\PycharmProjects\PyhtonPrograms\venv\lib\site-packages\goopylib\colours.py", line 2, in <module>
    from goopylib.math.Interpolations import *
  File "C:\Users\rithv\PycharmProjects\PyhtonPrograms\venv\lib\site-packages\goopylib\math\__init__.py", line 1, in <module>
    import goopylib.math.CBezierCurve
ImportError: DLL load failed while importing CBezierCurve: %1 is not a valid Win32 application.
```

Asus computer
Pycharm 2020.2.3

**Comment by BhavyeMathur** [comment]
Please go through [this](https://www.howtogeek.com/howto/21726/how-do-i-know-if-im-running-32-bit-or-64-bit-windows-answers/#:~:text=Checking%20Your%20Version%20of%20Windows,the%20%E2%80%9CSystem%20type%E2%80%9D%20entry.) website and tell me if you're running 32-bit or 64-bit Windows. Additionally, check your Python installation version by running the following code (you can copy-paste it to a new file):

```python
import struct
print(struct.calcsize("P") * 8)
```
Hopefully, the results from this will help me fix the bug.

**Comment by Rthvk** [comment]
I have a 32 bit windows. (I know this because the output was 32).

**Comment by BhavyeMathur** [comment]
and what did you get when you went to the website and checked your OS?

**Comment by Rthvk** [comment]
Screenshot:
![image](https://user-images.githubusercontent.com/72855820/96402457-8e79bc80-11f3-11eb-98ce-d14299721568.png)

**Comment by BhavyeMathur** [comment]
Your System Type would be under these specifications. Please send that.

**Comment by Rthvk** [comment]
I do not understand

**Comment by BhavyeMathur** [comment]
The screenshot you have sent is only part of the specifications. Please send the entire screen's screenshot so that I can see the System Type.  The System Type specification is located below the ones you have sent, so if I can see that, it'll help me solve the issue.

**Comment by r3ik3t** [comment]
I have 64 bit computer. it does not work for me either.

**Comment by r3ik3t** [comment]
it does not work for me either

**Comment by BhavyeMathur** [comment]
> I have 64 bit computer. it does not work for me either.

Can you run this code to check how many bits your Python runs on:

```python
import struct
print(struct.calcsize("P") * 8)
```

**Comment by r3ik3t** [comment]
yes 1 second

**Comment by r3ik3t** [comment]
my pycharm is not opening

**Comment by BhavyeMathur** [comment]
I see, try running the code in IDLE. Go to the taskbar, search up IDLE and open the first result that comes up. This should open a window and give you something which looks like this:

```python
>>>
```

Then, type:

```python
import struct
```

Press enter, then:

```python
struct.calcsize("P") * 8
```

And give me the output.

**Comment by r3ik3t** [comment]
![image](https://user-images.githubusercontent.com/72855906/96549476-28616800-12cd-11eb-8b2c-33d22e2081d0.png)

**Comment by r3ik3t** [comment]
here

**Comment by BhavyeMathur** [comment]
ok brilliant! So, you'll need to uninstall Python first. You have a 64-bit computer but a 32-bit Python installation. So after uninstalling, redownload Python MAKE SURE IT'S 64-BIT. https://www.python.org/downloads/windows/
![Screenshot 2020-10-20 at 12 13 01 PM](https://user-images.githubusercontent.com/67759015/96549803-9c037500-12cd-11eb-86bb-f91a7bd6290f.png)

Download the x86-64 executable installer and follow the installation instructions. This should fix the issue. After this, try running

```python
from goopylib.imports import *
```

in IDLE.

**Comment by r3ik3t** [comment]
will my earlier projects be saved??

**Comment by BhavyeMathur** [comment]
> will my earlier projects be saved??

Yes, they will remain where they were.

**Comment by r3ik3t** [comment]
![image](https://user-images.githubusercontent.com/72855906/96550727-d4578300-12ce-11eb-8389-33d0443124a7.png)

**Comment by r3ik3t** [comment]
should i press uninstall

**Comment by r3ik3t** [comment]
![image](https://user-images.githubusercontent.com/72855906/96551101-711a2080-12cf-11eb-9b71-4f438b8d96f5.png)

**Comment by r3ik3t** [comment]
it is not uninstalling

**Comment by r3ik3t** [comment]
ok i figured it out thanks

**Comment by BhavyeMathur** [comment]
Uninstall Python, not PyCharm. Try following this: https://medium.com/@itylergarrett.tag/my-story-about-uninstalling-python-3-7-pip-uninstall-python-lol-stop-f74618b1774d. And I suppose you'll have to reinstall PyCharm too.

**Comment by r3ik3t** [comment]
umm i do not know what i did but

![Uploading image.png…]()

**Comment by r3ik3t** [comment]
![Uploading image.png…]()

**Comment by r3ik3t** [comment]
should i check the "py."

**Comment by BhavyeMathur** [comment]
You commented without letting the image upload. Make sure to preview whatever you are posting before posting it.

**Comment by r3ik3t** [comment]
never mind i got it till here
![image](https://user-images.githubusercontent.com/72855906/96554094-ade81680-12d3-11eb-90c6-6f8b83d6e57f.png)

**Comment by r3ik3t** [comment]
never mind i got it till here

![image](https://user-images.githubusercontent.com/72855906/96554218-e0920f00-12d3-11eb-9580-4bdfbaff35e1.png)

**Comment by BhavyeMathur** [comment]
Ok great! have you uninstalled Python now?

**Comment by r3ik3t** [comment]
i do not know how do i do that
mu pycharm has opened and when i tried import goopylib thing you said this is what came
![image](https://user-images.githubusercontent.com/72855906/96554753-965d5d80-12d4-11eb-909a-c0f4171c8351.png)

**Comment by BhavyeMathur** [comment]
> Uninstall Python, not PyCharm. Try following this: https://medium.com/@itylergarrett.tag/my-story-about-uninstalling-python-3-7-pip-uninstall-python-lol-stop-f74618b1774d. 

Follow this please.

**Comment by r3ik3t** [comment]
Open your installer you downloaded off of python’s website. https://www.python.org/downloads/release/python-366/
You broke something, didn’t you?
Run the installer again.
Uninstall

huh?? what does that even mean

**Comment by BhavyeMathur** [comment]
Ok, it's fine, just do this:

1. Search up IDLE on your taskbar
2. Right Click it 
3. Click 'Uninstall Application'

This will open another window

4. Find Python (not PyCharm) and right click to click uninstall.

**Comment by r3ik3t** [comment]
thanks!

**Comment by r3ik3t** [comment]
![image](https://user-images.githubusercontent.com/72855906/96555768-ce18d500-12d5-11eb-9057-83a93c4c9cab.png)
wait umm did i download it ?

**Comment by BhavyeMathur** [comment]
Uninstall Python 3.8

**Comment by r3ik3t** [comment]
i uninstalled all python things and started again. I just installed these
![image](https://user-images.githubusercontent.com/72855906/96565976-aaa85700-12e2-11eb-96b4-f808d37509bb.png)

**Comment by r3ik3t** [comment]
the idle thing is still not working

**Comment by BhavyeMathur** [comment]
ok great, now open PyCharm, make sure you have goopylib installed and try importing it.

**Comment by r3ik3t** [comment]
i know it sounds dumb but my pycharm disappeared

**Comment by r3ik3t** [comment]
wait i found it will do it in sometime cause i need a password i do not have

**Comment by r3ik3t** [comment]
ok

**Comment by r3ik3t** [comment]
No Python at 'C:\Users\Rishi_2\AppData\Local\Programs\Python\Python38-32\python.exe'
this is what it says

**Comment by r3ik3t** [comment]
for all projects

**Comment by r3ik3t** [comment]
ok leave it i will ask you tomorrow

**Comment by BhavyeMathur** [comment]
Try using this website: https://www.jetbrains.com/help/pycharm/project-interpreter.html#70968 You'll also have to reinstall goopylib.

**Comment by r3ik3t** [comment]
i chose a system interpreter and now my other projects work but the goopylib one show this

![image](https://user-images.githubusercontent.com/72855906/96596777-b60f7880-130a-11eb-9def-2542034508cd.png)

**Comment by r3ik3t** [comment]
does that mean i install goopylib again

**Comment by r3ik3t** [comment]
i think i need too but do i put a check on this?
install to  users site package directory

**Comment by BhavyeMathur** [comment]
Choose Version 1.1.132a14 that's the latest one. And no, there's no need to check 'install to user's site package directory'.

**Comment by r3ik3t** [comment]
it is back here!

![image](https://user-images.githubusercontent.com/72855906/96597687-be1be800-130b-11eb-917e-610d36245ce2.png)

**Comment by BhavyeMathur** [comment]
are you sure you downloaded version 1.1.132a14?

**Comment by r3ik3t** [comment]
yes i can try again though

**Comment by BhavyeMathur** [comment]
please send me the screenshot of the screen where you are installing.

**Comment by BhavyeMathur** [comment]
ok great.

**Comment by r3ik3t** [comment]
ok now this is comng

![image](https://user-images.githubusercontent.com/72855906/96600602-bc075880-130e-11eb-8ac7-35df5f7e7b14.png)

**Comment by BhavyeMathur** [comment]
Could you please run it again and send me the error?

**Comment by r3ik3t** [comment]
![image](https://user-images.githubusercontent.com/72855906/96603010-3df88100-1311-11eb-93a2-ab39090cfea4.png)
here

**Comment by BhavyeMathur** [comment]
Update: The Issue has been resolved after using Python 3.8.6 rather than 3.9.
